projects/hip/docs/how-to/hip_porting_guide.rst

```diff
@@ -623,14 +623,13 @@ implementation, however it uses different parameters:
 
 The first parameter is the same as HIP's implementation, but
 ``MIN_BLOCKS_PER_MULTIPROCESSOR`` must  be converted to
-``MIN_WARPS_PER_EXECUTION``, which uses warps and execution units rather than
-blocks and multiprocessors. This conversion is performed automatically by
-:doc:`HIPIFY <hipify:index>`, or can be done manually with the following
+``MIN_WARPS_PER_EXECUTION_UNIT``, which uses warps and execution units rather than
+blocks and multiprocessors. This conversion can be done manually with the following
 equation.
 
 .. code-block:: cpp
 
-  MIN_WARPS_PER_EXECUTION_UNIT = (MIN_BLOCKS_PER_MULTIPROCESSOR * MAX_THREADS_PER_BLOCK) / warpSize
+  MIN_WARPS_PER_EXECUTION_UNIT = (MIN_BLOCKS_PER_MULTIPROCESSOR * MAX_THREADS_PER_BLOCK) / (warpSize * NUM_EUS_PER_CU)
```

Comment:
I don't see in this review where NUM_EUS_PER_CU is defined, but the terms CU and WGP have their own difficulties.